contrib/javaee.wildfly/licenseinfo.xml

```diff
@@ -47,4 +52,26 @@
         <license ref="W3C2" />
         <comment type="CATEGORY_B" />
     </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-app_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-service_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_7_3.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-web_4_0.dtd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/datasources_1_4.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-common_6_0.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/jboss-as-messaging-deployment_1_0.xsd</file>
+        <file>src/org/netbeans/modules/javaee/wildfly/resources/wildfly-messaging-activemq-deployment_1_0.xsd</file>
+        <license ref="LGPL-2.1" />
```

Comment:
I think LGPL is category X for Apache. Can you confirm the license?

Reply:
Yes
I can update some (like wildfly-messaging-activemq-deployment_1_0.xsd can be moved to ASL 2.0) I can also remove some of them but then we would loose support for old JBoss / WildFly versions

Reply:
It would be useful to comment out the LGPL stuff and replace what you can replace with a better license (like Apache 2.0).

Reply:
I'm almost done :)